control-plane-operator/controllers/hostedcontrolplane/kas/deployment.go

```diff
@@ -967,3 +966,12 @@ func buildKonnectivityVolumeClusterCerts(v *corev1.Volume) {
 		DefaultMode: pointer.Int32(0640),
 	}
 }
+
+func renderAuditLogScript(auditLogFilePath string) string {
+	var script = `#!/bin/bash
```

Comment:
Probably not, I was mainly just following what others do when calling `/bin/bash -c`like https://github.com/openshift/hypershift/blob/85148d58dc35663e1cb0ffe3fd1bd948cb765ad2/contrib/ci/hypershift-ci-1.yaml#L111-L119. There's quite a few more examples throughout the hypershift operator, so I thought it was common place for this repo.

Reply:
But in this case, the previous code doesn't do that.  So unless there's a good reason, please continue the pattern of the previous audit container setup.

Reply:
Previously, it was calling `tail` directly, so it couldn't have - if we're going to use `bash` it seems prudent to continue with `euo pipefail` - it will reduce the chance we introduce any of the common bugs those guard against.